Code/GraphMol/FileParsers/MaeMolSupplier.cpp

```diff
@@ -312,21 +312,27 @@ void set_atom_properties(Atom &atom, const mae::IndexedBlock &atom_block,
 
 void addAtoms(const mae::IndexedBlock &atom_block, RWMol &mol) {
   // All atoms are guaranteed to have these three field names:
-  const auto atomic_numbers = atom_block.getIntProperty(mae::ATOM_ATOMIC_NUM);
+  const auto atomicNumbers = atom_block.getIntProperty(mae::ATOM_ATOMIC_NUM);
   const auto xs = atom_block.getRealProperty(mae::ATOM_X_COORD);
   const auto ys = atom_block.getRealProperty(mae::ATOM_Y_COORD);
   const auto zs = atom_block.getRealProperty(mae::ATOM_Z_COORD);
 
   // atomic numbers, and x, y, and z coordinates
-  const auto size = atomic_numbers->size();
+  const auto size = atomicNumbers->size();
   auto conf = new RDKit::Conformer(size);
   conf->setId(0);
 
   PDBInfo pdb_info(atom_block);
 
   bool nonzeroZ = false;
   for (size_t i = 0; i < size; ++i) {
-    Atom *atom = new Atom(atomic_numbers->at(i));
+    auto atomicNumber = atomicNumbers->at(i);
+    if (atomicNumber == -2) {
+      // Maestro files use atomic number -2 to indicate a dummy atom.
+      atomicNumber = 0;
+    }
```

Comment:
The ancient code in atom.typ says:
```
Lone pairs get atomic number -1, dummy atoms -2. The values -3 and 0 are reserved for use by MacroModel; user attempts to specify these may or may not lead to desired effects, please avoid them.
```
I assume we should at lease throw an exception warning about lack of support for 0/-1/3? Or does it make sense to turn those all into rdkit atomicNumber=0?

Reply:
Let's throw a warning and remove the atoms one we have finished parsing the mol (just not adding would cause trouble with parsing the bonds, even more if one of these atoms was bonded).